README.md

```diff
@@ -25,14 +25,25 @@ and elegance of data integration via YAML to describe the data movement and tran
 The Flink CDC prioritizes efficient end-to-end data integration and offers enhanced functionalities such as 
 full database synchronization, sharding table synchronization, schema evolution and data transformation.
 
-![Flink CDC framework desigin](docs/static/fig/architecture.png)
+![Flink CDC framework design](docs/static/fig/architecture.png)
 
+### Quickstart Guide
 
+Flink CDC provides a CdcUp CLI utility to start a playground environment and run Flink CDC jobs.
+You will need to have a working Docker and Docker compose environment to use it.
+
+1. Run `git clone https://github.com/apache/flink-cdc.git --depth=1` to retrieve a copy of Flink CDC source code.
+2. Run `cd tools/cdcup/ && ./cdcup.sh init` to use the CdcUp tool to start a playground environment.
+3. Run `./cdcup.sh up` to initialize docker containers, and `./cdcup.sh pipeline` to submit a pipeline job.
```

Comment:
What about providing a default yaml file for mysql to Doris pipeline?
As we can check the result from Doris web ui directly.

Reply:
A viable yaml file should be generated after running `./cdcup.sh init`.